pkg/yang/yang.go

```diff
@@ -339,25 +339,26 @@ type Container struct {
 	Parent     Node         `yang:"Parent,nomerge"`
 	Extensions []*Statement `yang:"Ext"`
 
-	Anydata     []*AnyData   `yang:"anydata"`
-	Action      []*Action    `yang:"action"`
-	Anyxml      []*AnyXML    `yang:"anyxml"`
-	Choice      []*Choice    `yang:"choice"`
-	Config      *Value       `yang:"config"`
-	Container   []*Container `yang:"container"`
-	Description *Value       `yang:"description"`
-	Grouping    []*Grouping  `yang:"grouping"`
-	IfFeature   []*Value     `yang:"if-feature"`
-	Leaf        []*Leaf      `yang:"leaf"`
-	LeafList    []*LeafList  `yang:"leaf-list"`
-	List        []*List      `yang:"list"`
-	Must        []*Must      `yang:"must"`
-	Presence    *Value       `yang:"presence"`
-	Reference   *Value       `yang:"reference"`
-	Status      *Value       `yang:"status"`
-	Typedef     []*Typedef   `yang:"typedef"`
-	Uses        []*Uses      `yang:"uses"`
-	When        *Value       `yang:"when"`
+	Anydata      []*AnyData      `yang:"anydata"`
+	Action       []*Action       `yang:"action"`
+	Anyxml       []*AnyXML       `yang:"anyxml"`
+	Choice       []*Choice       `yang:"choice"`
+	Config       *Value          `yang:"config"`
+	Container    []*Container    `yang:"container"`
+	Description  *Value          `yang:"description"`
+	Grouping     []*Grouping     `yang:"grouping"`
+	IfFeature    []*Value        `yang:"if-feature"`
+	Leaf         []*Leaf         `yang:"leaf"`
+	LeafList     []*LeafList     `yang:"leaf-list"`
+	List         []*List         `yang:"list"`
+	Must         []*Must         `yang:"must"`
+	Notification []*Notification `yang:"notification"`
```

Comment:
Just confirming, did these `notification` lines come in from rebasing some other commits?

Reply:
No, it came from me implementing a YANG module that defined notifications not at the top-level, and realizing that support was lacking here.

I can split it out into a separate PR if you prefer.